src/test/cli.build.test.ts

```diff
@@ -27,6 +27,21 @@ describe('Dev Containers CLI', function () {
 
 	describe('Command build', () => {
 
+		it('should correctly configure the image name to push from --image-name with --push true', async () => {
+			let success = false;
+			const testFolder = `${__dirname}/configs/example`;
+			try {
+				await shellExec(`${cli} build --workspace-folder ${testFolder} --image-name demo:v1 --push true`);
```

Comment:
Is `--push` risky as the local developer or CI could be signed in to a default registry that would be pushed to?

Reply:
this test case does not have docker login setup, so it will fail and we are just checking the image is correctly populated in the error message or not. Yes, it is risky if the local developer or CI is configured with the default registry.